src/test/java/codeu/controller/ConversationServletTest.java

```diff
@@ -148,6 +148,29 @@ public void testDoPost_ConversationNameTaken() throws IOException, ServletExcept
     Mockito.verify(mockResponse).sendRedirect("/chat/test_conversation");
   }
 
+  @Test
+  public void testDoPost_NewConversation_NoTags() throws IOException, ServletException {
+    Mockito.when(mockRequest.getParameter("conversationTitle")).thenReturn(TEST_CONVERSATION);
+    Mockito.when(mockSession.getAttribute("user")).thenReturn(TEST_USERNAME);
+    Mockito.when(mockSession.getAttribute("password")).thenReturn(TEST_PASSWORD);
+    Mockito.when(mockSession.getAttribute("about")).thenReturn(TEST_ABOUT);
+    Mockito.when(mockRequest.getParameter("tags")).thenReturn(null);
```

Comment:
Is setting `mockRequest.getParameter("tags")` to `null` what you want? Do you want to actually not even set it?

I'm not clear if returning `null` is the equivalent behavior to it not being present in the parameters at all

Reply:
I just checked and if there are no new tags in the input field an empty string is returned rather than `null`, so I adjusted the test accordingly